src/cfml/system/modules_app/server-commands/commands/server/stop.cfc

```diff
@@ -21,18 +21,28 @@ component aliases="stop" {
 	 * @forget Remove the directory information from disk
 	 * @all Stop ALL running servers
 	 * @verbose Show raw output of stop command
+	 * @local  Stop servers with webroot matching the current directory
 	 **/
 	function run(
 		string name,
 		string directory,
 		String serverConfigFile,
 		boolean forget=false,
 		boolean all=false,
-		boolean verbose=false ){
-
+		boolean verbose=false,
+		boolean local=false
+	){
+		var localOnly = arguments.local;
```

Comment:
Why do we need a copy of this variable?  Just use `arguments.local`

Reply:
fixed